core/src/main/java/edu/wpi/grip/core/operations/Operations.java

```diff
@@ -67,6 +76,52 @@
                 new OperationMetaData(WatershedOperation.DESCRIPTION, () -> new WatershedOperation(isf, osf)),
                 new OperationMetaData(ThresholdMoving.DESCRIPTION, () -> new ThresholdMoving(isf, osf)),
 
+                new OperationMetaData(
+                        OperationDescription.builder()
+                                .name("Crop")
+                                .summary("Crop an image")
+                                .icon(Icon.iconStream("grip"))
+                                .build(),
+                        templateFactory.create(
+                                SocketHints.Inputs.createMatSocketHint("src", false),
+                                SocketHints.Inputs.createPointSocketHint("p1", false),
+                                SocketHints.Inputs.createPointSocketHint("p2", false),
+                                SocketHints.Inputs.createMatSocketHint("dst", true),
```

Comment:
No. But the Crop operation should be in its own class file like every other custom operation.